flintrock/flintrock.py

```diff
@@ -297,6 +305,21 @@ def launch(
         raise UnsupportedProviderError(provider)
 
 
+def get_last_commit_sha(git_repository: str):
```

Comment:
I think this signature is better expressed as follows:

```python
def get_latest_commit(github_repository: str):
```

The `_latest_` matches the name of the option value `'latest'`, and the `github_repository` emphasizes that this method is not for any git repository.